resource/storage.go

```diff
@@ -155,7 +155,7 @@ func (s storageWrapper) MultiGet(ctx context.Context, ids []interface{}) (items
 }
 
 // Find tries to use storer MultiGet with some pattern or Find otherwise
-func (s storageWrapper) Find(ctx context.Context, lookup *Lookup, page, perPage int) (list *ItemList, err error) {
+func (s storageWrapper) Find(ctx context.Context, lookup *Lookup, page, perPage, offset int) (list *ItemList, err error) {
```

Comment:
`offset` can't be used in combination with `page`. I've also said this in the docs. so `page` is still needed. in fact, `page` is always 1 anyway when `offset` is used. so `pagePage` would still be descriptive even then.